.github/workflows/hls.yml

```diff
@@ -39,4 +45,4 @@ jobs:
             owner: context.repo.owner,
             repo: context.repo.repo,
             body: "```\n" + comment + "```"
-          });
\ No newline at end of file
+          });
```

Comment:
Noise

Reply:
Sure, but most text editors insert trailing newlines automatically, so I think it makes sense to have it be there, rather than having to keep in mind that you do not add it